bindings/python/crocoddyl/multibody/costs/cost-sum.cpp

```diff
@@ -24,14 +24,17 @@ BOOST_PYTHON_MEMBER_FUNCTION_OVERLOADS(CostModelSum_calc_wraps, CostModelSum::ca
 
 void exposeCostSum() {
   // Register custom converters between std::map and Python dict
+  typedef boost::shared_ptr<CostItem> CostItemPtr;
   typedef boost::shared_ptr<CostDataAbstract> CostDataPtr;
-  bp::to_python_converter<std::map<std::string, CostItem, std::less<std::string>,
-                                   std::allocator<std::pair<const std::string, CostItem> > >,
-                          map_to_dict<std::string, CostItem> >();
+
+  bp::register_ptr_to_python<boost::shared_ptr<CostItem> >();
+  bp::to_python_converter<std::map<std::string, CostItemPtr, std::less<std::string>,
+                                   std::allocator<std::pair<const std::string, CostItemPtr> > >,
+                          map_to_dict<std::string, CostItemPtr> >();
```

Comment:
I think you need to add the `false` flag, i.e.
```cpp
  bp::to_python_converter<std::map<std::string, CostItemPtr, std::less<std::string>,
                                   std::allocator<std::pair<const std::string, CostItemPtr> > >,
                          map_to_dict<std::string, CostItemPtr, false> >();
```

This should fix the issue. Let me know if this is not the case

Reply:
Thank you, this fixed it! I pushed the changes now and it should pass the buildserver soon.